src/context/json.ts

```diff
@@ -12,23 +12,17 @@ type JSONContextOptions = {
  * res(json('Some string'))
  * res(json([1, '2', false, { ok: true }]))
  */
-export const json = <BodyTypeJSON, BodyTypeString extends string>(
+export const json = <BodyTypeJSON>(
   body: BodyTypeJSON,
   { merge = false }: JSONContextOptions = {},
-): ResponseTransformer<BodyTypeString> => {
+): ResponseTransformer<BodyTypeJSON> => {
```

Comment:
This removes any typing quirk I'd hope.

Reply:
Can you please explain more on how this works? I see that you've renamed 

```diff
- BodyType
+ BodyTypeJSON
```

But I'm certainly missing something here...